cl/opinion_page/templates/court.html

```diff
@@ -82,71 +64,37 @@ <h1>Decisions for the Tennessee Court of Workers' Compensation Claims and Tennes
+        {% if results.paginator.count > 0 %}
+          {% include "includes/search_result.html" with results=results type_override=SEARCH_TYPES.OPINION simple=True %}
+          <p class="left">
+            <a href="/?court_{{ pk }}=on&order_by=dateFiled+desc"
                class="btn btn-default btn-lg v-offset-above-2"
                rel="nofollow"
-            >See Recent Opinions</a>
+            >See More Recent Opinions</a>
           </p>
         {% else %}
           <p>No cases yet.</p>
         {% endif %}
-
       </div>
       <div class="col-xs-4">
+        <h3>About the Court</h3>
+          <p>The Supreme Judicial Court of Maine is the highest court in the state of Maine. It is made up of a chief justice and six associate justices who are appointed by the governor and confirmed by the Maine Senate. The court has general appellate jurisdiction and is responsible for interpreting the state's laws and constitution. It also has the power to issue advisory opinions on matters of law to the governor and the state's legislature.</p>
+        <hr>
         <h3>Neutral Citations</h3>
-        <p>Beginning on January 1st, 2020 the Tennessee Court of Workers’ Compensation Claims and the Appeals Board will accept pleadings, briefs, and other filings that include <a href="{% url "faq" %}#explain-neutral-citations">vendor neutral citations</a> to prior decisions as discussed below. All previous decisions have been retroactively assigned such a citation. These citations are created for each decision by the court itself instead of by a third party publisher. As a result, they can be cited immediately when the decision is published by the court.
-        </p>
-        <h4>Format</h4>
-        <p>Citations to the Claims Court take the form:</p>
-        <pre>[YEAR] TN WC [#]</pre>
-        <p>Citations to the Appeals Board take the form:</p>
-        <pre>[YEAR] TN WC App. [#]</pre>
-        <p>In both cases, the format begins with the year, then an abbreviation for the court, and finishes with a serial number indicating the decision number.</p>
+          <p>A neutral citation is a standardized legal citation that is used to uniquely identify court opinions and other legal documents. Neutral citations are used to provide a consistent and reliable way to cite legal materials in the United States.</p> <p>They typically include the year of the decision, the court or tribunal that issued the decision, and a unique identifier for the specific decision. Neutral citations can be used in addition to traditional legal citations and can make it easier for legal researchers to locate and access specific legal materials. </p>
+        <p>Citations to the {{ court }} take the form:</p>
+        <pre>[YEAR] ME [#]</pre>
+        <p>In this cases, the format begins with the year, then an abbreviation for the court (ME), and finishes with a serial number indicating the decision number.</p>
```

Comment:
Please see this message: https://github.com/freelawproject/courtlistener/pull/2449#issuecomment-1374272229